.travis.yml

```diff
@@ -48,5 +48,5 @@ matrix:
   exclude:
     - scala: 2.10.7
       env: PLATFORM=js  SBT_PARALLEL=true  WORKERS=1 DEPLOY=true SCALAJS_VERSION=1.0.0-M3
-    - scala: 2.13.0-M4
-      env: PLATFORM=js  SBT_PARALLEL=true  WORKERS=1 DEPLOY=true SCALAJS_VERSION=1.0.0-M3
+    - scala: 2.13.0-M5
+      env: PLATFORM=js  SBT_PARALLEL=true  WORKERS=1 DEPLOY=true SCALAJS_VERSION=0.6.25
```

Comment:
@SethTisue updated 0.6.25 in Travis, but missed the value in`plugins.sbt`, as well

Reply:
Actually, this is an exclude rule, so I'm wrong about the first part, but I think I'm right about the second part.  It's important because I don't think Rickard uses Travis to deploy

Reply:
@ashawley nice catch! https://github.com/rickynils/scalacheck/pull/435